src/main/java/org/mockito/invocation/MatchableInvocation.java

```diff
@@ -0,0 +1,70 @@
+package org.mockito.invocation;
+
+import org.mockito.ArgumentMatcher;
+
+import java.util.List;
+
+/**
+ * <code>MatchableInvocation</code> wraps {@link Invocation} instance
+ * and holds argument matchers associated with that invocation.
+ * It is used during verification process:
+ *
+ * <pre class="code"><code class="java">
+ *   mock.foo();   // <- invocation
+ *   verify(mock).bar();  // <- matchable invocation
+ * </code></pre>
+ *
+ * @since @since@
+ */
+public interface MatchableInvocation extends DescribedInvocation {
```

Comment:
I prefer Matchable because technically speaking it's also used for matching invocations for stubbing.